lib/Window.js

```diff
@@ -4,6 +4,7 @@ var EventTarget = require('./EventTarget');
 var Location = require('./Location');
 var sloppy = require('./sloppy');
 var utils = require('./utils');
+var DOMParserWrapper = require("./DOMParser");
```

Comment:
Why is this `DOMParserWrapper` and not just `DOMParser`?  None of the other classes have `...Wrapper` suffixes.